server/app/models/ApplicationStatusesModel.java

```diff
@@ -0,0 +1,55 @@
+package models;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.ebean.annotation.DbJson;
+import io.ebean.annotation.WhenCreated;
+import java.time.Instant;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import play.data.validation.Constraints;
+import services.program.StatusDefinitions;
+
+/** An EBean class that stores all the statuses related to programs. */
+@Entity
+@Table(name = "application_statuses")
+public class ApplicationStatusesModel extends BaseModel {
+  private static final long serialVersionUID = 1L;
+  @Constraints.Required private String programName;
+
+  @Constraints.Required @DbJson private StatusDefinitions statusDefinitions;
```

Comment:
Re my comment about renaming, I'm not sure what the best name for this column would be. Maybe renaming to `status_definition_set` or `status_set`?

Reply:
This would be the actually be the statusDefinition object itself. My understanding was a row in ApplicationStatuses table is called a StatusSet

Reply:
It's a json list of `StatusDefinition` objects, right? Not a singular `StatusDefinition` object?

So `StatusDefinitions` works, but I thought it might be confusing to name the get `getStatusDefinitions()` when you're getting a singular thing, hence trying to come up with a name for the list so "the list of status definitions" is a singular thing.

Reply:
so to clarify my thought: The json list of status definitions stored in a particular row would be the "status set".

Reply:
nvm, per my comment on the main PR